resource-directory/service/deviceMetadataProjection.go

```diff
@@ -130,7 +131,7 @@ func (p *deviceMetadataProjection) Handle(ctx context.Context, iter eventstore.I
 				return err
 			}
 			if err := p.data.HandleDeviceMetadataUpdatePending(ctx, &e); err != nil {
-				return err
+				goto nextEvent
```

Comment:
continue doesn't work for a for loop when called from an inner switch?

Reply:
fixed